frontend/src/components/ShootDetails/GShootKubeconfig.vue

```diff
@@ -141,24 +195,66 @@ export default {
     isGardenloginType () {
       return this.type === 'gardenlogin'
     },
+    isStaticKubeconfigType () {
+      return this.type === 'token'
+    },
+    isAdminKubeconfigType () {
+      return this.type === 'adminkubeconfig'
```

Comment:
We should split this component into different one for each type. Let's do it at least for this one. If we have reuse should be done via composable.